docker/nginx/api.nginx.conf

```diff
@@ -15,6 +15,14 @@ http {
     proxy_next_upstream error;
 
     if ($http_x_forwarded_proto != "https") {
+      set $https_redirect "1";
+    }
+
+    if ($http_user_agent != "ELB-HealthChecker/2.0") {
+      set $https_redirect "${https_redirect}1";
+    }
+
+    if ($https_redirect = "11") {
```

Comment:
I basically copied it from here https://www.digitalocean.com/community/questions/conditional-within-an-if-in-nginx
Apparently nginx does not support multiple conditions in an if statement.
I hate nginx BTW...